mfr/server/static/js/mfr.child.hypothesis.js

```diff
@@ -62,7 +62,23 @@
             window.document.head.appendChild(script);
             window.document.body.classList.add('show-hypothesis');
             hypothesisLoaded = true;
-        });
 
+            var sidePanelOpened = false;
+            var sendAnalyticsIfExpanded = function (expanded) { 
+                if (expanded && !sidePanelOpened) {
```

Comment:
Probably a dumb question about `hypothesis` configuration. What is `expanded` (or `layout.expanded` a few lines  below more specifically)?

Reply:
`layout.expanded` denotes whether the hypothesis side panel is expanded when `onLayoutChanged()` is called.